core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java

```diff
@@ -154,6 +156,27 @@ public final AbstractLineSender boolColumn(CharSequence name, boolean value) {
         return field(name, value);
     }
 
+    public AbstractLineSender fieldTimestamp(CharSequence name, long value) {
```

Comment:
this is unnecessary. 

I'm keeping the old `field()` methods only for the sake of backward compatibility. 
my thinking is that after a few releases - when users hopefully migrate to the new `Sender` interface - we would remove the old methods entirely.